snap/snapcraft.yaml

```diff
@@ -0,0 +1,30 @@
+name: dss
+summary: a ML development and experimentation environment manager
+description: |
+     The DSS is a tool that allows users of workstations to deploy
+     and manage development and experimentation environments
+     in systems with GPU support.
+adopt-info: dss
+base: core22
+confinement: strict
+
+parts:
+  dss:
+    plugin: python
+    source: .
+    override-pull: |
+      craftctl default
+      # Comment out after the data-science-stack repository has tags for versioning the snap
+      # craftctl set version=$(git describe --tags --abbrev=10)
+      craftctl set version="0.1"
```

Comment:
An easy way to avoid needing to do this would be to create an initial tag like "0.0" as a pre-release.  Not important though.